update-schema.main.kts

```diff
@@ -120,7 +125,11 @@ fun run() {
     executeCommand("git", "push", "origin", branch)
 
     authenticateGithubCli()
-    executeCommand("gh", "pr", "create", "-f")
+    
+    val baseBranchArgument = baseBranch?.let {
+        "-B $it"
+    } ?: ""
+    executeCommand("gh", "pr", "create", "-t", prTitle, "-b", prBody, baseBranchArgument)
```

Comment:
Doesn't adding a "" extra argument going to confuse `gh` there is no shell there so it's going to be passed as an extra arg.